tests/Test.proj

```diff
@@ -0,0 +1,5 @@
+﻿<Project Sdk="Microsoft.Build.Traversal/2.0.24">
+  <ItemGroup>
+    <ProjectReference Include="**\*.*proj" />
+  </ItemGroup>
+</Project>
```

Comment:
Nit : seems to be missing trailing end line.

Btw do we run some formatter on the code?

Reply:
:point_right:  https://github.com/redis/NRedisStack/blob/master/.github/workflows/linter.yaml

Reply:
Wonder why this was not caught by the linter then

Reply:
this file is something i introduced just to run tests one assembly at a time.
so it is not an artifact in sln file in root. 

also seem like `dotnet format` does not cover it anyway.  
```
dotnet format ./tests/Test.proj
Could not format '/home/atakavci/workspace/redis/nredisstack/NRedisStack/tests/Test.proj'. Format currently supports only C# and Visual Basic projects.
```
funny it is valid and accepted by `dotnet test`

Reply:
Sure, that's fine
I just wanted to make sure the liner is working, not a big issue, thanks for checking this